### PR TITLE
[test] Update MpiT Check

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

**Comment by jgphpc** [comment]
@teojgo what do you think of the failure in  https://jenkins.cscs.ch/blue/organizations/jenkins/ReframeCI/detail/ReframeCI/5472/pipeline  ?

**Comment by jgphpc** [comment]
retest this please

**Comment by jgphpc** [comment]
> Please upload with the Codecov repository upload token to resolve issue.

File: cscs-checks/prgenv/mpi_t.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 
 @rfm.simple_test
-class MpiTCheck(rfm.RegressionTest):
+class MpiT_Check(rfm.RegressionTest):
```
Comment (jjotero) [inline cscs-checks/prgenv/mpi_t.py:12]:
```suggestion
class mpit_check(rfm.RegressionTest):
```

**Review by jjotero — COMMENTED** [review]
Other than the test name (should be all lowercase with underscores), looks good!

**Review by teojgo — APPROVED** [review]

**Comment by jgphpc** [comment]
merging for c4k release

**Pull request merged**